<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="2" failed="0" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="8866" started-at="2016-06-03T16:15:23Z" finished-at="2016-06-03T16:15:32Z">
    <groups>
      <group name="create">
        <method signature="PlpPages.beforeClass(java.lang.String)[pri:0, instance:citi.thankyou.PlpPages@2812cbfa]" name="beforeClass" class="citi.thankyou.PlpPages"/>
      </group> <!-- create -->
    </groups>
    <test name="Default test" duration-ms="8866" started-at="2016-06-03T16:15:23Z" finished-at="2016-06-03T16:15:32Z">
      <class name="citi.thankyou.PlpPages">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:citi.thankyou.PlpPages@2812cbfa]" name="beforeTest" is-config="true" duration-ms="6" started-at="2016-06-03T12:15:23Z" finished-at="2016-06-03T12:15:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="FAIL" signature="beforeClass(java.lang.String)[pri:0, instance:citi.thankyou.PlpPages@2812cbfa]" name="beforeClass" is-config="true" duration-ms="0" started-at="2016-06-03T12:15:23Z" finished-at="2016-06-03T12:15:23Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefoxMAC]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
	at citi.thankyou.PlpPages.beforeClass(PlpPages.java:118)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:175)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:107)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="SKIP" signature="beforeClass(java.lang.String)[pri:0, instance:citi.thankyou.PlpPages@2812cbfa]" name="beforeClass" duration-ms="0" started-at="2016-06-03T12:15:32Z" finished-at="2016-06-03T12:15:32Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[firefoxMAC]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="FAIL" signature="afterMethod_takeScreenshot(org.testng.ITestResult)[pri:0, instance:citi.thankyou.PlpPages@2812cbfa]" name="afterMethod_takeScreenshot" is-config="true" duration-ms="0" started-at="2016-06-03T12:15:32Z" description="take screenshot on fails" finished-at="2016-06-03T12:15:32Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=beforeClass status=SKIP method=PlpPages.beforeClass(java.lang.String)[pri:0, instance:citi.thankyou.PlpPages@2812cbfa] output={null}]]]>
              </value>
            </param>
          </params>
          <exception class="java.io.IOException">
            <message>
              <![CDATA[Destination '/failed' directory cannot be created]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.IOException: Destination '/failed' directory cannot be created
	at org.apache.commons.io.FileUtils.copyFile(FileUtils.java:1085)
	at org.apache.commons.io.FileUtils.copyFile(FileUtils.java:1038)
	at citi.thankyou.PlpPages.afterMethod_takeScreenshot(PlpPages.java:288)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:786)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.IOException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod_takeScreenshot -->
        <test-method status="SKIP" signature="f()[pri:0, instance:citi.thankyou.PlpPages@2812cbfa]" name="f" duration-ms="0" started-at="2016-06-03T12:15:32Z" finished-at="2016-06-03T12:15:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- f -->
        <test-method status="FAIL" signature="afterMethod_takeScreenshot(org.testng.ITestResult)[pri:0, instance:citi.thankyou.PlpPages@2812cbfa]" name="afterMethod_takeScreenshot" is-config="true" duration-ms="0" started-at="2016-06-03T12:15:32Z" description="take screenshot on fails" finished-at="2016-06-03T12:15:32Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=f status=SKIP method=PlpPages.f()[pri:0, instance:citi.thankyou.PlpPages@2812cbfa] output={null}]]]>
              </value>
            </param>
          </params>
          <exception class="java.io.IOException">
            <message>
              <![CDATA[Destination '/failed' directory cannot be created]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.IOException: Destination '/failed' directory cannot be created
	at org.apache.commons.io.FileUtils.copyFile(FileUtils.java:1085)
	at org.apache.commons.io.FileUtils.copyFile(FileUtils.java:1038)
	at citi.thankyou.PlpPages.afterMethod_takeScreenshot(PlpPages.java:288)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:786)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.IOException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod_takeScreenshot -->
        <test-method status="SKIP" signature="afterClass()[pri:0, instance:citi.thankyou.PlpPages@2812cbfa]" name="afterClass" is-config="true" duration-ms="0" started-at="2016-06-03T12:15:32Z" finished-at="2016-06-03T12:15:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
        <test-method status="SKIP" signature="afterTest()[pri:0, instance:citi.thankyou.PlpPages@2812cbfa]" name="afterTest" is-config="true" duration-ms="0" started-at="2016-06-03T12:15:32Z" finished-at="2016-06-03T12:15:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- citi.thankyou.PlpPages -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
